samples/apps/copilot-chat-app/webapi/SemanticKernelExtensions.cs

```diff
@@ -43,6 +43,8 @@ internal static IServiceCollection AddSemanticKernelServices(this IServiceCollec
                 .WithLogger(sp.GetRequiredService<ILogger<IKernel>>())
                 .WithMemory(sp.GetRequiredService<ISemanticTextMemory>())
                 .WithConfiguration(sp.GetRequiredService<KernelConfig>())
```

Comment:
We don't need the kernel config anymore now that we're using the kernel builder for service registration. This will cause an InvalidOperationException error. 
```suggestion
```